Phonebook.Frontend/src/app/modules/profile-picture/components/profile-picture/profile-picture.component.html

```diff
@@ -1,7 +1,6 @@
 <div
   id="image-container"
   (mouseenter)="onMouseenter()"
-  (mouseleave)="onMouseleave()"
```

Comment:
removing this will cause the dialog to not close automatically in the table view